insights/combiners/tests/test_sys_vmbus_devices.py

```diff
@@ -0,0 +1,72 @@
+import doctest
+
+from insights.combiners import sys_vmbus_devices
+from insights.parsers.sys_vmbus import SysVmbusDeviceID, SysVmbusClassID
+from insights.tests import context_wrap
+
+
+DEVICE_ID_1 = """
+{47505500-0001-0000-3130-444531444234}
+""".strip()
+
+CLASS_ID_1 = """
+{44c4f61d-4444-4400-9d52-802e27ede19f}
+""".strip()
+
+DEVICE_ID_2 = """
+{4487b255-b88c-403f-bb51-d1f69cf17f87}
+""".strip()
+
+CLASS_ID_2 = """
+{3375baf4-9e15-4b30-b765-67acb10d607b}
+""".strip()
+
+DEVICE_ID_3 = """
```

Comment:
This `DEVICE_ID_3` is not used in any test cases, better remove it or add it to the test.

Reply:
@xiangce Good catch. Removed unused vars.